nbs/example.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<p>Do you want to take a crack at the annotations?  I usually put the image in Google slides and put shapes with numbers or something like that.   I can also do it too if you don't have time, just let me know!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/811/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>